ethereumetl/streaming/postgres_tables.py

```diff
@@ -39,8 +39,8 @@
     Column('state_root', String),
     Column('receipts_root', String),
     Column('miner', String),
-    Column('difficulty', Numeric(38)),
-    Column('total_difficulty', Numeric(38)),
+    Column('difficulty', String),
+    Column('total_difficulty', String),
```

Comment:
I think this change is unnecessary

Reply:
It should actually be Numeric(38, 9)

Reply:
Yes, I thought we changed the other way around but the data should be Numeric, fixed.